presto-docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,92 @@
+====================
+Hudi connector
+====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+--------
+Overview
+--------
+
+Apache Hudi is a rich platform to build streaming data lakes with incremental data pipelines
```

Comment:
Let's summarize the overview in one or two sentences. This doc is not meant to tell the user how Hudi works, but how to use the connector. It's sufficient to provide a link to Hudi website for people who are interested in the underlying techniques.

Reply:
done